Feature/more controls over custom trigger (#244)
MatthewHerbst/react-to-print: README.md

|    **`bodyClass`**    | `string?`   | Class to pass to the print window body                                                                                     |
|    **`suppressErrors`**    | `boolean?`   | When passed, prevents `console` logging of errors

### PrintContext

> In case you don't want to specify `trigger` directly within other props but you rather want to gain some extra control over it, you could make use of PrintContext. By using PrintContext you can get direct access to the `handlePrint` method which triggers printing action.

Reviewer: ```suggestion
If you need extra control over printing and don't want to specify `trigger` directly, `PrintContext` allows you to gain direct access to the `handlePrint` method which triggers the print action.
```

### [useReactToPrint]

> For the functional components, there is a useReactToPrint hook, which accepts an object with the same configuration props as `<ReactToPrint />` itself and returns a handlePrint function, by calling which you can trigger printing action.

Reviewer: ```suggestion
For functional components, use the `useReactToPrint` hook, which accepts an object with the same configuration props as `<ReactToPrint />` returns a `handlePrint` function which when called will trigger the print action.
```

    </div>
  );
};

```

> ### Calling from functional components with [useReactToPrint](https://reactjs.org/docs/hooks-intro.html)

Reviewer: ```suggestion
### Calling from functional components with `useReactToPrint` [hook](https://reactjs.org/docs/hooks-intro.html)
```
